client/components/chart/style.scss

```diff
@@ -96,7 +96,7 @@
 			width: 100%;
 			text-transform: uppercase;
 			font-size: 11px;
-			color: $core-grey-dark-100;
+			color: $core-grey-dark-300;
 			opacity: 0.6;
```

Comment:
@Aljullu This still only has a contrast ratio of `2.3` – when I gave that color value I didn't see that it was partially transparent. If you remove the opacity it should be fine 👍  I use [Shade](http://halogensoftworks.com/shade/) to check this, I don't know if there's a good linux alternative though.

Reply:
Wow, I completely missed the opacity too! I will fix that tomorrow.

I see Chrome has a similar tool in the devtools, I will make sure to use that one in the future. :slightly_smiling_face: